README.md

```diff
@@ -53,12 +53,21 @@ count-files -h
 count-files --help
 ```
 
-By default, the program counts or searches for files recursively in current working directory and all of its subdirectories. Any hidden files or folders will be ignored.
-The names of extensions are case insensitive by default. The results for `ini` and `INI` will be the same.
-
+By default, the program counts or searches for files recursively in current working directory and all of its subdirectories.  
+For fully supported operating systems (Linux, macOS, iOS, Windows), any hidden files or folders are ignored by default. For other operating systems in which Python can be run, this option to include/exclude hidden files is not available. And as a result, all existing files will be included.  
+The names of extensions are case insensitive by default. The results for `ini` and `INI` will be the same.  
 Optionally, you can pass it a path to the directory to scan, choose non-recursive counting or searching, process the file extensions with case-sensitive mode and enable search or counting in hidden files and folders.  
-You can get additional information about the size of each found file and see a short preview for text files.  
-See more about [CLI arguments](https://countfiles.readthedocs.io/en/latest/howtouse.html#cli-arguments).
+
+When counting all files by extension (default table):  
```

Comment:
I am not sure the following lines should be included in this file. In my opinion, they fit better the full documentation that the simpler description we are trying to keep here.